README.md

```diff
@@ -609,10 +610,6 @@ project, please read the [contributing guide](https://github.com/mercedes-benz/o
 Please read our [Code of Conduct](https://github.com/mercedes-benz/daimler-foss/blob/master/CODE_OF_CONDUCT.md)
 as it is our base for interaction.
 
-## License
```

Comment:
Remove license from table of content

Reply:
There still is a License section. This one was just the duplicate...